tests/core/test_interface.py

```diff
@@ -38,11 +38,11 @@ def setUp(self):
     def test_init(self):
         assert self.Cu_GB1.rotation_angle == approx(123.74898859588858)
         assert self.Cu_GB1.vacuum_thickness == approx(1.5)
-        assert self.Cu_GB2.rotation_axis == [1, 2, 3]
+        assert_allclose(self.Cu_GB2.rotation_axis, [1, 2, 3])
         assert_allclose(self.Cu_GB1.ab_shift, [0.0, 0.0])
         assert_allclose(self.Cu_GB2.ab_shift, [0.2, 0.2])
-        assert self.Cu_GB1.gb_plane == [1, 3, 1]
-        assert self.Cu_GB2.gb_plane == [1, 2, 3]
+        assert_allclose(self.Cu_GB1.gb_plane, [1, 3, 1])
+        assert_allclose(self.Cu_GB2.gb_plane, [1, 2, 3])
```

Comment:
i would prefer the old code. no need to use `assert_allclose` with integers. better to have more sensitive tests that fail if we expect something to be integers and somehow they become floats

Reply:
Okay sure f3cabe8d1a7017ef38050d4620ae359783b36c98. But then we need to change its type to `tuple` :)

Reply:
yes, feel free to remove the `del type_name` and `del ppdesc` lines

Reply:
Thanks for the confirmation :)